pkg/cloudscale/objectstorage.go

```diff
@@ -132,9 +132,14 @@ func (o *ObjectStorage) GetMetrics(ctx context.Context, billingDate time.Time) (
 		salesOrder := o.salesOrder
 		if salesOrder == "" {
 			appuioManaged = false
+			if bucket.Organization == "" {
+				// in cases that our VSHN services are using buckets, then Organization is not set, we must default it to "vshn"
+				// we can't set it in cluster as for customers as then we might run into scheduling issues
+				bucket.Organization = "vshn"
```

Comment:
I'm not sure I would go with this solution, we need to make sure all namespaces have an organization instead of setting the billing to vshn. What if there was a bug somewhere and the namespace is not owned by vshn with empty billing org label, later we find out it was a customer namespace meanwhile everything was billed for vshn?

Reply:
We can't set this label or we'll mess up the scheduling in our cluster. This seems like a fair compromise.

Reply:
what do you mean mess up scheduling?

Reply:
we use different set of nodes and if we just yolo set org label on namespace, then our pods inside this namespace won't be able to schedule

Reply:
We need to discuss this. It might come up that it's not even our problem.